### ISSUE TITLE
zetteldeft-org-show-graph is very needed 

### ISSUE THREAD

**Issue by vlnn** [EFLS/zetteldeft#54]
org-roam-show-graph provides an easy way to overview and navigate rather complex structures of interlinked notes. Wouldn't it be great to have same workflow in zetteldeft? At the moments it's possible either insert the dot data, or save pdf -- but not integrate it into the emacs as shown here https://github.com/jethrokuan/org-roam/raw/master/doc/images/org-roam-graph.gif

**Comment by EFLS** [comment]
Sure looks convenient, but outside of my scope at the moment, both in terms of time and elisp skill. But who knows, maybe in a few months time.

Another source of inspiration is `org-graph-view`: https://github.com/alphapapa/org-graph-view

Contributions in the form of PRs or ideas on how to implement something like this are of course welcome.

**Comment by AtomicNess123** [comment]
Has there been anything put into motion regarding graphing the links between notes?

**Comment by EFLS** [comment]
> Has there been anything put into motion regarding graphing the links between notes?

No changes yet. Basic graphing still relies on graphviz. See https://www.eliasstorms.net/zetteldeft/zetteldeft.html#visuals

I’m open to suggestions for improvements, or even PRs, but my skills are limited.

**Comment by AtomicNess123** [comment]
Thanks. Do I understand correctly that the graphviz functionality is embedded already with the instalation of zetteldeft? Thanks.

**Comment by EFLS** [comment]
Yes, those functions are already part of Zetteldeft. It's rather crude and simply generates relevant Org-mode source blocks that you can then execute to generate a PDF drawing. The creation of that graph relies on graphviz, so you'll need to have the graphviz executable installed (https://graphviz.org/)
